### PR TITLE
Update YAML files for 1.3.0

### PR DESCRIPTION
**Pull request by scholzj**
This PR tries to update the YAML files with examples, installation files and OperatorHub integration for AMQ Streams 1.3.0.

### CODE REVIEW

File: operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1076 @@
...
+
+    AMQ Streams also supports advanced operations such as adding or removing disks in Kafka brokers or resizing the persistent volumes (where supported by the infrastructure).
+
+    ### Documentation
+
+    Documentation for the current release can be found on Red Hat [website](https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/index).
+    
+  keywords: ['amq', 'amq-streams', 'kafka', 'messaging', 'kafka-connect', 'kafka-streams', 'data-streaming', 'data-streams', 'streaming', 'streams']
+  version: 1.3.0
+  replaces: amqstreams.v1.3.0
```
Comment (tombentley) [inline operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml]: Surely this replaces 1.2.0?

File: operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,1076 @@
...
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+        - apiGroups:
+          - kafka.strimzi.io
+          resources:
+          - kafkatopics
```
Comment (tombentley) [inline operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml]: Don't we need `kafkatopics/status` here too, for delegation?

File: operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,1076 @@
...
+            metadata:
+              labels:
+                name: amq-streams-cluster-operator
+                strimzi.io/kind: cluster-operator
+            spec:
+              serviceAccountName: strimzi-cluster-operator
+              containers:
+              - name: cluster-operator
+                image: registry.redhat.io/amq7/amq-streams-operator:1.3.0
+                imagePullPolicy: IfNotPresent
```
Comment (tombentley) [inline operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml]: I thought we were going to get rid of this?

File: operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml (hunk 2 above)
Comment (scholzj) [reply operator/modules/olm/manifests/1.3.0/amq-streams.v1.3.0.clusterserviceversion.yaml]: I didn't touched the RBAC. You didn't updated them?

**Comment by scholzj** [comment]
@tombentley The review comments should be fixed now

**Pull request merged**